inst/extdata/gcam-china/CCS_provincial_data.csv

```diff
@@ -0,0 +1,1678 @@
+# File: CCS_provincial_data.csv
```

Comment:
Create a new issue regarding sharing the original data of CCS cost curves.

Reply:
？

Reply:
@sypnl do you mind opening the issue and reference this PR? Other than that, is this ready to merge?

Reply:
Done! Please go ahead and merge it. Thank you! @rynacui

Reply:
Impressive. Nice job! 👏